scripts/remove/lidarr.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
-user=$(cut -d: -f1 < /root/.master.info)
 systemctl disable --now -q lidarr
 
-rm -rf /home/$user/Lidarr
-rm -rf /home/$user/.config/Lidarr/
```

Comment:
We are transitioning to *not* removing user data? Imo, Ombi is a bug and given the confusion around this that people generally expect that the application will be entirely removed if you use a remover.

Reply:
Either way this will be refactored based on the prowlarr PR to provide the option to purge or not

Reply:
L5 is necessary (not sure this comment references that Line)
L6, is bringing this in line with the other arrs as they are in master right now

https://github.com/swizzin/swizzin/blob/master/scripts/remove/radarr.sh
https://github.com/swizzin/swizzin/blob/master/scripts/remove/sonarr.sh